CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# v1.24.0
+### Notable Changes
+* Support clustered allocation with ext4 filesystems. This allows developers to enable [torn write prevention](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/storage-twp.html) on their dynamically provisioned volumes to improve the performance of I/O-intensive relational database workloads. ([#1706](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1706), [@AndrewSirenko](https://github.com/AndrewSirenko))
+* Add volume limits for m7a, c7i, r7a, r7iz instance families ([#1742](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1742), [@torredil](https://github.com/torredil))
+* Add volume attachment limits for c7a instance family ([#1776](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1776), [@torredil](https://github.com/torredil))
```

Comment:
np: Can we combine these into a single entry? ie:

Add volume limits for m7a, c7i, c7a, r7a, r7iz, instance families (#1742, #1776)

Reply:
Sure, I'll link to both PRs like so: ([#1742],[#1776], [@torredil])